### ISSUE TITLE
Assertion failed: type == NN_EP_STOPPED

### ISSUE THREAD

**Issue by gonzus** [nanomsg/nanomsg#75]
[Testing main branch with fresh pull as of 2013-07003 21:15 GMT]

When running the (Java-based) {local,remote}_lat test with parameters "tcp://127.0.0.1:6677 100 1000", I got the following assertion. The test does work most of the time.

Assertion failed: type == NN_EP_STOPPED (..\..\src\core\sock.c:742)

**Comment by singhan-chen** [comment]
I have the same error with nanomsg-0.5.
```
Assertion failed: src == NN_SOCK_SRC_EP && type == NN_EP_STOPPED (src/core/sock.c:830)
Aborted
```
It failed `tests/ipc_shutdown`.

I did a quick test with https://github.com/wirebirdlabs/featherweight-nanomsg,
they seems fixed this issue already.

**Comment by gdamore** [comment]
Can someone test the current git repo?  Is this still an isuse?

**Comment by gdamore** [comment]
Actually, I'm certain that this got fixed with 

commit 7555f49e83ff414133f77e46389f8e5632eeea1f
Author: Garrett D'Amore <garrett@damore.org>
Date:   Fri Oct 23 08:01:22 2015 -0700

    fixes #469 assertion failure in ipc_shutdown (really)

Closing it.
